## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 481517`
Commit: `466961814b55ed015e883ee7a29643c9eff58322`

---
### `aarch64-darwin`
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>python314Packages.compressai</li>
    <li>python314Packages.compressai.dist</li>
    <li>python314Packages.skrl</li>
    <li>python314Packages.skrl.dist</li>
    <li>python314Packages.spacy-loggers</li>
    <li>python314Packages.spacy-loggers.dist</li>
    <li>python314Packages.wandb</li>
    <li>python314Packages.wandb.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 229 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>flattenReferencesGraph</li>
    <li>flattenReferencesGraph.dist</li>
    <li>jadx</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python313Packages.arviz-plots</li>
    <li>python313Packages.arviz-plots.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python313Packages.bumps</li>
    <li>python313Packages.bumps.dist</li>
    <li>python313Packages.chart-studio</li>
    <li>python313Packages.chart-studio.dist</li>
    <li>python313Packages.cltk</li>
    <li>python313Packages.cltk.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.energyflow.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.holistic-trace-analysis</li>
    <li>python313Packages.holistic-trace-analysis.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.pymoo</li>
    <li>python313Packages.pymoo.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.sasmodels</li>
    <li>python313Packages.sasmodels.dist</li>
    <li>python313Packages.scikit-tda</li>
    <li>python313Packages.scikit-tda.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>python314Packages.arviz-plots</li>
    <li>python314Packages.arviz-plots.dist</li>
    <li>python314Packages.bumps</li>
    <li>python314Packages.bumps.dist</li>
    <li>python314Packages.chart-studio</li>
    <li>python314Packages.chart-studio.dist</li>
    <li>python314Packages.dash</li>
    <li>python314Packages.dash-bootstrap-components</li>
    <li>python314Packages.dash-bootstrap-components.dist</li>
    <li>python314Packages.dash-bootstrap-templates</li>
    <li>python314Packages.dash-bootstrap-templates.dist</li>
    <li>python314Packages.dash.dist</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.experiment-utilities.dist</li>
    <li>python314Packages.explorerscript</li>
    <li>python314Packages.explorerscript.dist</li>
    <li>python314Packages.holistic-trace-analysis</li>
    <li>python314Packages.holistic-trace-analysis.dist</li>
    <li>python314Packages.hvplot</li>
    <li>python314Packages.hvplot.dist</li>
    <li>python314Packages.igraph</li>
    <li>python314Packages.igraph.dist</li>
    <li>python314Packages.intake</li>
    <li>python314Packages.intake.dist</li>
    <li>python314Packages.iplotx</li>
    <li>python314Packages.iplotx.dist</li>
    <li>python314Packages.itables</li>
    <li>python314Packages.itables.dist</li>
    <li>python314Packages.kmapper</li>
    <li>python314Packages.kmapper.dist</li>
    <li>python314Packages.leidenalg</li>
    <li>python314Packages.leidenalg.dist</li>
    <li>python314Packages.neurokit2</li>
    <li>python314Packages.neurokit2.dist</li>
    <li>python314Packages.niaarm</li>
    <li>python314Packages.niaarm.dist</li>
    <li>python314Packages.plotly</li>
    <li>python314Packages.plotly.dist</li>
    <li>python314Packages.reflex</li>
    <li>python314Packages.reflex-chakra</li>
    <li>python314Packages.reflex-chakra.dist</li>
    <li>python314Packages.reflex.dist</li>
    <li>python314Packages.sasmodels</li>
    <li>python314Packages.sasmodels.dist</li>
    <li>python314Packages.scikit-tda</li>
    <li>python314Packages.scikit-tda.dist</li>
    <li>python314Packages.synergy</li>
    <li>python314Packages.synergy.dist</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python314Packages.wandb</summary>
<pre>Setting COVERAGE_FILE to /nix/var/nix/builds/nix-build-python3.14-wandb-0.21.4.drv-0/b/source/.coverage
=============================== warnings summary ===============================
../../../../../../store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: 29 warnings
  /nix/store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The &#x27;repr&#x27; attribute with value False was provided to the `Field()` function, which has no effect in the context it was used. &#x27;repr&#x27; is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
    warnings.warn(

../../../../../../store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: 29 warnings
  /nix/store/2p20k2cy8cv4bnh90lj5lma3fzwf17r5-python3.14-pydantic-2.12.4/lib/python3.14/site-packages/pydantic/_internal/_generate_schema.py:2249: UnsupportedFieldAttributeWarning: The &#x27;frozen&#x27; attribute with value True was provided to the `Field()` function, which has no effect in the context it was used. &#x27;frozen&#x27; is field-specific metadata, and can only be attached to a model field using `Annotated` metadata or by assignment. This may have happened because an `Annotated` type alias using the `type` statement was used, or if the `Field()` function was attached to a single member of a union type.
    warnings.warn(

tests/unit_tests/test_wandb_settings_old.py::test__global_path_env_var_exists_and_is_writable
  /nix/var/nix/builds/nix-build-python3.14-wandb-0.21.4.drv-0/b/source/tests/unit_tests/test_wandb_settings_old.py:33: PytestWarning: Value of environment variable WANDB_CONFIG_DIR type should be str, but got PosixPath(&#x27;/nix/var/nix/builds/nix-build-python3.14-wandb-0.21.4.drv-0/b/pytest-of-_nixbld10/pytest-0/popen-gw9/test__global_path_env_var_exis0/config&#x27;) (type: PosixPath); converted to str implicitly
    monkeypatch.setenv(&quot;WANDB_CONFIG_DIR&quot;, config_dir)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/unit_tests/test_torch.py::test_watch_parameters_torch_jit[parameters] - AttributeError: &#x27;Linear&#x27; object has no attribute &#x27;__annotations__&#x27;. Did you mean: &#x27;__annotate_func__&#x27;?
FAILED tests/unit_tests/test_torch.py::test_watch_parameters_torch_jit[all] - AttributeError: &#x27;Linear&#x27; object has no attribute &#x27;__annotations__&#x27;. Did you mean: &#x27;__annotate_func__&#x27;?
FAILED tests/unit_tests/test_torch.py::test_watch_graph_torch_jit - AttributeError: &#x27;Net&#x27; object has no attribute &#x27;__annotations__&#x27;. Did you mean: &#x27;__annotate_func__&#x27;?
= 3 failed, 1801 passed, 142 skipped, 3 xfailed, 5 xpassed, 59 warnings in 104.81s (0:01:44) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/plotly/default.nix
+++ b/pkgs/development/python-modules/plotly/default.nix
@@ -38,14 +38,14 @@
 
 buildPythonPackage rec {
   pname = "plotly";
-  version = "6.5.0";
+  version = "6.5.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "plotly";
     repo = "plotly.py";
     tag = "v${version}";
-    hash = "sha256-JWb5bAu74j63E5tp8wmLjuWZqFAMpkg8utxM74VaGqA=";
+    hash = "sha256-7rMatpaZvHuNPpiXR5eUHultqNnLER1iW+GR3dwgkyo=";
   };
 
   postPatch = ''

